docker/entrypoint-celery-cron-worker.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+exec celery --workdir=/home/app/vaas \
+  -A vaas.settings worker \
+  --loglevel=DEBUG \
+  --concurrency=1 \
+  -Q cron_queue
```

Comment:
Why do we spawn workers for each queue? As far as I know, we could run celery multi to spawn all workers in a single container.